frontend/src/modules/about-us/components/SidebarSection/constants.ts

```diff
@@ -0,0 +1,57 @@
+export const sidebarItems = [
```

Comment:
พวก constants ใช่เป็น `SNAKE_UPPER_CASE` ดีกว่าครับ เวลาอ่านโค้ดจะได้รู้ได้เลยว่าเป็น constants
```suggestion
export const SIDEBAR_ITEMS = [
```